index.html

```diff
@@ -7,13 +7,53 @@
     <title>CodersCamp2020 | Star Wars API QUIZ</title>
     <link rel="stylesheet" href="styles/App.css"/>
 
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-          rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+    <script src="https://kit.fontawesome.com/8495d163a8.js" crossorigin="anonymous"></script>
+
 </head>
+
 <body>
 
-<div id="swquiz-app">
-</div>
+    <main>
+        <div class="logoImgSettings">
+            <div id="logo"></div>
+            <div id="img" class="boxShadow"></div>
+            <button id="settingsButton" class="button hoverCursor"><h3> Settings </h3></button>
+        </div>
+
+        <div class="gameArea">
+            <div id="changeGameMode" class="bgdiv boxShadow">
+                <button id="gameMode1" class="button gameModeButton hoverCursor"><h2>People</h2></button>
```

Comment:
you should write some code to check user selection and send it to App.js, maybe gameChanger.component class and generate html elements from there, instead of hard-coding them in index.html